src/Utils/dbConnection.ts

```diff
@@ -0,0 +1,16 @@
+import { getConnectionOptions, createConnection } from "typeorm";
+import { NODE_ENV } from "./config";
```

Comment:
add `NODE_ENV` to the `.env.example` file

Reply:
wip